content/_guides/helpers.md

```diff
@@ -0,0 +1,165 @@
+
+```irc
+11:49 -- Mode #windows [+Ccnptjfr]
+11:49 -- Channel #windows created on Wed, 19 May 2021 07:36:22
+11:49 -- [#windows] $~a quieted by copper.libera.chat on Mon, 14 Jun 2021 10:49:19
+11:49 -- [#windows] End of Channel Quiet List
+```
+
+If the user needs to be identified, and they are not, they might need
+reminding. You can do this when redirecting them politely.
+
+```irc
+<You> I don't know much about Windows, but the people in #windows will! You will need to be registered to talk in there. If you have an account, see '/msg nickserv help identify'. If not, see '/msg nickserv help register' :)
+```
+
+## They didn't read the topic
+
+IRC can be a bit overwhelming when first joining a channel, especially for new
+people. In addition, some clients may not show the whole topic on join.
+Instead of just telling the person to `/topic` or "read the topic", it's
+easier and more friendly to tell them the answer even if you copy and paste
+directly from the topic.
+
+## Avoid answering people with bots
+
+If you always use a bot trigger to answer a person, the channel atmosphere
+will be cold and robotic. Using bots to answer people will make them feel like
+they're in a call center phone queue. Ick!
+
+It is better to take the time to write something yourself; people can tell
+the difference and will pay attention to you better than to a bot. Some people
+think factoid bots are necessary because people are impatient and leave
+quickly. This almost always is due to a lack of response, not a lack of
+solution. If you greet them and ask them to wait, they probably will.
+
+If _you_ need the factoid to remind you of the information, and the bot does
+not give factoids in PM, explain as you're doing it. This also teaches them
+how to use the bot. Also, don't just leave them hanging with the answer! Check
+it yourself to ensure it has the information you think it does, and prompt the
+user to confirm if it covers what they want.
+
+## When to stop engaging
+
+When a situation requires intervention and your own attempts to diffuse it do
+not seem to be working, it is sometimes best to stand down and stop
+contributing to the noise. Instead, try to [seek out a channel operator](faq#how-to-find-out-who-runs-a-channel)
+quietly, such as with private messages or in an `-ops` channel if there is
+one.
+
+If you know that someone else around is an Operator, and they seem to be
+paying attention to the channel while the disruption is occurring, they might
+already be talking to the person in private. Check with the Operator if you're
+unsure before also trying to intervene.
+
+When an Operator starts handling the situation, continuing to engage with the
+person who is misbehaving may derail the Operator's efforts, as the person may
+become defensive at being "ganged up on".
+
+If necessary, use the `/ignore` functionality in your client to stop
+contributing. You don't want to cause the situation to escalate, or you might
+also end up in trouble.
```

Comment:
Something about this wording feels hostile to me "don't escalate or there will be CONSEQUENCES" *thunder roars in the background*

Not sure if I have a better suggestion, maybe something along the lines of "there may be instances where stepping in might lead to the situation escalating and you becoming a part of the issue". But not sure that sounds any better